src/stream/src/common/log_store_impl/in_mem.rs

```diff
@@ -245,7 +245,7 @@ impl LogReader for BoundedInMemLogStoreReader {
 }
 
 impl LogWriter for BoundedInMemLogStoreWriter {
-    async fn init(
+    async fn init_after_yield_barrier(
```

Comment:
IIUC, renaming allows us to review all usages explicitly in this PR. Since this has been done now, shall we consider renaming them back?

Reply:
Same for other occurrences.